webserver/courses/views.py

```diff
@@ -122,3 +124,62 @@ def get_course_info(request, id):
             ),
             status=400,
         )
+
+@api_view(['POST'])
+def add_image_to_course(request, id):
+    if request.user.is_anonymous:
+        return Response(
+            data=dict(
+                reason='You must be logged in to join a course.',
+            ),
+            status=400,
+        )
+    if request.FILES.get('image') == None:
+        return Response(
+            data=dict(
+                reason='Image file is required.',
+            ),
+            status=400,
+    )
+    try:
+        course = Course.objects.get(id=id)
+        image_file = request.FILES.get('image')
+        labeled_image = LabeledImage.objects.create(name=image_file.name, owner=request.user, image=image_file)
+        course.images.add(labeled_image)
+        serialized_image = LabeledImageSerializer(labeled_image)
+    except Course.DoesNotExist:
+        return Response(
+            data=dict(
+                reason='No course with the provided code was found. Please make sure you entered it correctly.',
+            ),
+            status=400,
+        )
+    return Response(
+        serialized_image.data, 
+        status=201
+    )
```

Comment:
No need to return the data here
```suggestion
    return Response()
```